clang/test/CXX/drs/cwg22xx.cpp

```diff
@@ -169,6 +169,20 @@ B b;
 // since-cxx11-error@-1 {{call to implicitly-deleted default constructor of 'B'}}
 //   since-cxx11-note@#cwg2273-B {{default constructor of 'B' is implicitly deleted because base class 'A' has a deleted default constructor}}
 //   since-cxx11-note@#cwg2273-A {{'A' has been explicitly marked deleted here}}
+
```

Comment:
Should we claim that cwg2273 is only supported in clang 21? @Endilll

Reply:
Yes, with the updated test it's evident that Clang 20 and earlier versions don't exhibit the correct behavior.
Don't forget to run `clang/www/make_cxx_dr_status` script after changing the status on line 158.